README.md

```diff
@@ -17,9 +17,28 @@
 
 # Pangea Python Mono Repo
 
-The Pangea repository for all things Python
+This is the Pangea repository for all things Python
 
-## Pangea Python SDK
 
-[Pangea SDK](https://github.com/pangeacyber/pangea-python/tree/main/packages/pangea-sdk)
-[Pangea SDK examples](https://github.com/pangeacyber/pangea-python/tree/main/examples)
+## Pangea Python Tools
+
+This is a list of tools or useful stuff that you will found out in this repository and their links
+
+- Pangea Python SDK [Click here](/packages/pangea-sdk) 
+- List of Pangea Python SDK examples per service [Click here](/examples) 
+
+
+## Repository structure
+
+|- **README.md**: This readme file [Click here](/README.md)  
+|- **CHANGELOG.md**: SDK version changes tracking [Click here](/CHANGELOG.md)  
+|- **examples**: SDK examples folder [Clik here](/examples)  
```

Comment:
can we just have the hyperlink be [SDK examples folder](/examples)? I think that'd be cleaner

Reply:
I like that way too, but we were talking yesterday and we arrive to the conclusion to make it more explicit that user can click and they'll go redirected